virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume-block.adoc

```diff
@@ -11,9 +11,9 @@ file.
 
 [WARNING]
 ====
-Cloning operations between different volume modes are not supported. The `volumeMode` values must match in both the source and target specifications.
+Cloning operations between different volume modes is supported, such as cloning from a persistent volume (PV) with `volumeMode: Block` to a PV with `volumeMode: Filesystem`.
 
-For example, if you attempt to clone from a persistent volume (PV) with `volumeMode: Block` to a PV with `volumeMode: Filesystem`, the operation fails with an error message.
+However, you can only clone between different volume modes if they are of the kubevirt content-type.
```

Comment:
As we discussed via Slack, I think this needs some formatting and a slight tweak to better reflect the API reference. One suggestion: "...if they have `contentType: "kubevirt"`. Alternatively, "...if they have the `"kubevirt"` content type."

Reply:
(reference: https://kubevirt.io/cdi-api-reference/v1.15.0/definitions.html#_v1alpha1_datavolumespec)